### DISCUSSION TITLE
homebrew bintray SSL issue / no downloads possible macOS 12.2.2

### DISCUSSION THREAD

**Discussion by gmkey** [Homebrew/discussions#962] [Everyday usage]
Hi as  of today I can't seem to download any homebrew packages;

e.g.: downloading icu4c
curl: (35) error:1400410B:SSL routines:CONNECT_CR_SRVR_HELLO:wrong version number
Error: Failed to download resource "icu4c"
Download failed: https://homebrew.bintray.com/bottles/icu4c-68.2.big_sur.bottle.tar.gz
==> Upgrading libomp 11.0.1 -> 11.1.0
==> Downloading https://homebrew.bintray.com/bottles/libomp-11.1.0.big_sur.bottle.tar.gz

when I try to reach the package using firefox, i get this message:
SSL_ERROR_NO_CYPHER_OVERLAP

I've tried to download the packages from another machine (linux box - no homebrew installed). 

wget https://homebrew.bintray.com/bottles/libomp-11.1.0.big_sur.bottle.tar.gz
--2021-02-28 16:56:31--  https://homebrew.bintray.com/bottles/libomp-11.1.0.big_sur.bottle.tar.gz
Herleiden van homebrew.bintray.com (homebrew.bintray.com)... 65.9.83.81, 65.9.83.33, 65.9.83.117, ...
Connect to homebrew.bintray.com (homebrew.bintray.com)|65.9.83.81|:443... connected
GnuTLS: A packet with illegal or unsupported version was received.
Can't establish an SSL-connection

so it seems a serverside issue:

**Comment by carlocab** [comment]
https://github.com/Homebrew/discussions/discussions/961
https://github.com/Homebrew/brew/issues/10739

**Comment by richleigh** [comment]
I'm assuming there isn't a nice trivial way to just say 'skip the TLS handshake and caveat-emptor style get to the binaries'?

**Reply by Bo98** [reply]
No, it's an outage that affects plain HTTP too.

**Reply by richleigh** [reply]
Ah, my fail. From the message I though it was a TLS cert problem, not an endpoint problem.

**Reply by Bo98** [reply]
Yeah, if the server isn't establishing the TLS connection properly (or incorrectly returning a plaintext error message over HTTPS - which I suspect it might be doing), errors can often be cryptic.

**Reply by jake1412** [reply]
Would anyone know what this outage is caused by or how long would it last?

**Reply by gromgit** [reply]
Seems to be fixed as I write this.
